bundles/org.openhab.binding.ws980wifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+			</channel>
+			<channel id="tempWindChill" typeId="system.outdoor-temperature">
+				<label>Wind Chill Temperature</label>
+			</channel>
+			<channel id="heatIndex" typeId="system.outdoor-temperature">
+				<label>Heat Index Temperature</label>
+			</channel>
+			<channel id="humidityInside" typeId="system.atmospheric-humidity">
+				<label>Humidity Inside</label>
+			</channel>
+			<channel id="humidityOutside" typeId="system.atmospheric-humidity">
+				<label>Humidity Outside</label>
+			</channel>
+			<channel id="pressureAbsolut" typeId="system.barometric-pressure">
+				<label>Pressure Absolut</label>
+			</channel>
+			<channel id="pressureRelative" typeId="system.barometric-pressure">
+				<label>Pressure Relative</label>
+			</channel>
+			<channel id="windDirection" typeId="system.wind-direction">
+				<label>Wind Direction</label>
+			</channel>
+			<channel id="windSpeed" typeId="system.wind-speed">
+				<label>Wind Speed</label>
+			</channel>
+			<channel id="windSpeedGust" typeId="system.wind-speed">
+				<label>Wind Gust Speed</label>
+			</channel>
+			<channel id="rainLastHour" typeId="rain">
+				<label>Rain Last Hour</label>
+			</channel>
+			<channel id="rainLastDay" typeId="rain">
+				<label>Rain Last Day</label>
+			</channel>
+			<channel id="rainLastWeek" typeId="rain">
+				<label>Rain Last Week</label>
+			</channel>
+			<channel id="rainLastMonth" typeId="rain">
+				<label>Rain Last Month</label>
+			</channel>
+			<channel id="rainLastYear" typeId="rain">
+				<label>Rain Last Year</label>
+			</channel>
+			<channel id="rainTotal" typeId="rain">
+				<label>Rain Total</label>
+			</channel>
+			<channel id="lightLevel" typeId="lux">
+				<label>Light Level</label>
+			</channel>
+			<channel id="uvRaw" typeId="solarRadiation">
+				<label>UV Radiation</label>
+			</channel>
+			<channel id="uvIndex" typeId="uvIndex">
+				<label>UV Index</label>
+			</channel>
+		</channels>
+
+		<properties>                                                                    <!-- static properties, that are the same for all instances -->
+			<property name="vendor">ELV</property>
+			<property name="modelId">WS980WiFi</property>
+		</properties>
+
+		<representation-property>macAddress</representation-property>                   <!-- eindeutiger Identifier fuer jede WS -->
+
+		<config-description>                                                            <!-- konfigurierbare Parameter, indiv. fuer jede WS -->
+			<parameter name="host" type="text" required="true">
+				<label>Hostname</label>
+				<description>The hostname/IP address the device is bound to, e.g. 192.168.0.2</description>
+				<context>network-address</context>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="macAddress" type="text" required="true">
+				<label>MAC Address</label>
+				<description>The unique MAC address of the device, e.g. 00:10:FA:6E:38:4A</description>
+				<context>network-address</context>
```

Comment:
This should not be context network-addres. It's a hint for the ui to possible use a specific ui element.

Reply:
I do not understand what you want me to do?